target/linux/lantiq/image/Makefile

```diff
@@ -664,6 +664,18 @@ define Device/avm_fritz7360sl
 endef
 TARGET_DEVICES += avm_fritz7360sl
 
+define Device/avm_fritz7362sl
+  $(Device/AVM)
+  $(Device/NAND)
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 49152k
+  DEVICE_DTS := FRITZ7362SL
+  DEVICE_TITLE := AVM FRITZ!Box 7362 SL
+  DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic kmod-usb-dwc2 fritz-tffs
+  SUPPORTED_DEVICES += FRITZ7362SL
```

Comment:
@danijelt You can drop this line, since you're adding support for the new device which is added automatically for you in

https://github.com/openwrt/openwrt/blob/75fdf3ba01e9379f4301ccbaf3eeedf0923076f1/target/linux/lantiq/image/Makefile#L64

Reply:
Done.